package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -73,21 +65,20 @@ def search(self, atoms, radius, level='A'):
 
         Parameters
         ----------
-        atoms : AtomGroup, MDAnalysis.core.groups.Atom
-          list of atoms
+        atoms : AtomGroup, MDAnalysis.core.groups.AtomGroup
+          AtomGroup object
         radius : float
           Radius for search in Angstrom.
         level : str
           char (A, R, S). Return atoms(A), residues(R) or segments(S) within
           *radius* of *atoms*.
         """
         unique_idx = []
-        if isinstance(atoms, Atom):
-            positions = atoms.position.reshape(1, 3)
-        else:
-            positions = atoms.positions
-        
-        pairs = capped_distance(positions, self.atom_group.positions,
+        try:
+            position = atoms.positions
+        except AttributeError:
```

Comment:
good solution, just add a comment saying that in this case case, atoms is just a single Atom.